client/apps/edit/components/content/section_list/index.jsx

```diff
@@ -86,11 +97,24 @@ export class SectionList extends Component {
           sections={sections}
           index={-1}
           key={1}
-          isEditing={editingIndex !== null}
+          isEditing={activeSection !== null}
           firstSection
           isDraggable={false}
         />
       </div>
     )
   }
 }
+
+const mapStateToProps = (state) => ({
+  ...state
+})
```

Comment:
Its probably best to pluck out exactly what pieces of state you need from the code for explicitness, but either way is really fine. That said, this can be simplified to:

```javascript
const mapStateToProps = state => state
````